### ISSUE TITLE
Cannot create a backup now

### ISSUE THREAD

**Issue by marcatsmci** [nextcloud/all-in-one#754]
### Steps to reproduce
1.Create Backup button

### Expected behavior 
Backup done...

### Actual behavior 
```
Database export failed the last time. Most likely was the export time not high enough.
Cannot create a backup now.
Please report this to https://github.com/nextcloud/all-in-one/issues. Thanks!

```
#### Nextcloud AIO version 
1.3.0
#### Current channel 
Stable

**Comment by marcatsmci** [comment]
ok, after 2 (!) mastercontainer updates, backups are ok now...

**Comment by szaimen** [comment]
Yes it was an issue in 1.3.0 and fixed with 1.3.1. See also https://github.com/nextcloud/all-in-one/issues/749
